sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -20,6 +22,35 @@ export class ApplicationOfferingChangeRequestService {
     private readonly applicationOfferingChangeRequestRepo: Repository<ApplicationOfferingChangeRequest>,
   ) {}
 
+  /**
+   * Gets an eligible application that can be requested for application
+   * offering change.
+   * @param locationId location id.
+   * @param options method options:
+   * - `applicationId` specific eligible application to be retrieved.
+   * @returns eligible application.
+   */
+  async getEligibleApplications(
```

Comment:
Should we remove the **s** at the end: `getEligibleApplication`

Reply:
If we remove the "s" at the end this will no longer be an overload on the first method. I do not think that it is worth but I would do it if that is the Team's preference.

Reply:
I am ok with having the name as it is looking at the benefits of having overriding here. IMO having the applicationId in options is making it more clear.

Reply:
I am okay with what we have now

Reply:
I am okay too

Reply:
Ok, as per the Team's decision, I am keeping it as it is right now.

Reply:
Sounds good 👍